Mathlib/Topology/MetricSpace/Pseudo/Defs.lean

```diff
@@ -99,16 +99,17 @@ private theorem dist_nonneg' {α} {x y : α} (dist : α → α → ℝ)
     _ = 2 * dist x y := by rw [two_mul, dist_comm]
   nonneg_of_mul_nonneg_right this two_pos
 
-/-- Pseudo metric and Metric spaces
-
-A pseudo metric space is endowed with a distance for which the requirement `d(x,y)=0 → x = y` might
-not hold. A metric space is a pseudo metric space such that `d(x,y)=0 → x = y`.
-Each pseudo metric space induces a canonical `UniformSpace` and hence a canonical
-`TopologicalSpace` This is enforced in the type class definition, by extending the `UniformSpace`
-structure. When instantiating a `PseudoMetricSpace` structure, the uniformity fields are not
-necessary, they will be filled in by default. In the same way, each (pseudo) metric space induces a
-(pseudo) emetric space structure. It is included in the structure, but filled in by default.
--/
+/-- A pseudo metric space is a topological space and uniform space equipped with a metric `dist`,
```

Comment:
```suggestion
/-- A pseudometric space is a topological space and uniform space equipped with a metric `dist`,
```
and everywhere else in the new docstrings

Reply:
I am not sure I agree here

Reply:
It does seem like "pseudometric" is used far more than "pseudo metric" is, informally speaking.